salt/kube-apiserver/init.sls

```diff
@@ -12,6 +12,24 @@ include:
          cn = grains['nodename'],
          o = pillar['certificate_information']['subject_properties']['O']) }}
 
+/etc/pki/kube-apiserver-proxy-client.crt:
+  file.managed:
+    - contents_pillar: api:admission_webhook:cert
+    - user: kube
+    - group: kube
+    - mode: 644
+    - require:
+      - pkg: kube-apiserver
+
+/etc/pki/kube-apiserver-proxy-client.key:
+  file.managed:
+    - contents_pillar: api:admission_webhook:key
+    - user: kube
+    - group: kube
+    - mode: 644
```

Comment:
i think the private key should have `444` as we have for all other keys in https://github.com/kubic-project/salt/blob/b00d7641640c0394e1ffdd875728a60d44ba2d41/salt/_macros/certs.jinja#L60